src/pages/Admin/Users/Add/index.jsx

```diff
@@ -0,0 +1,320 @@
+              >
+                Re-enter password
+              </InputContainer>
+
+              <p className="font-demi my-1">E-mail Notification</p>
+              <InputContainer
+                type="checkbox"
+                checked={checkboxes.enote}
+                name="enote"
+                id="enote"
+                onChange={handleChange}
+              >
+                Check to enable email notification when upload scan completes.
+              </InputContainer>
+
+              <p className="font-demi mt-1">Default upload visibility</p>
+              <CommonFields
+                accessLevel={addUserData.accessLevel}
+                handleChange={handleChange}
+              />
+
+              <label htmlFor="agents" className="font-demi my-1">
+                Agents selected by default when uploading
+                {Object.keys(checkboxes).map((key) => {
+                  if (key !== "enote")
+                    return (
+                      <InputContainer
+                        key={key}
+                        type="checkbox"
+                        checked={checkboxes[key]}
+                        name={key}
+                        id={key}
+                        onChange={handleChange}
+                      >
+                        {(() => {
+                          switch (key) {
+                            case "Check_agent_copyright":
+                              return "Copyright/Email/URL/Author Analysis";
+                            case "Check_agent_ecc":
+                              return "ECC Analysis, scanning for text fragments potentially relevant for export control";
+                            case "Check_agent_keyword":
+                              return "Keyword Analysis";
+                            case "Check_agent_mimetype":
+                              return "MIME-type Analysis (Determine mimetype of every file. Not needed for licenses or buckets)";
+                            case "Check_agent_monk":
+                              return "Monk License Analysis, scanning for licenses performing a text comparison";
+                            case "Check_agent_nomos":
+                              return "Nomos License Analysis, scanning for licenses using regular expressions";
+                            case "Check_agent_ojo":
+                              return "Ojo License Analysis, scanning for licenses using SPDX-License-Identifier";
+                            case "Check_agent_pkgagent":
+                              return "Package Analysis (Parse package headers)";
+                            case "Check_agent_reso":
+                              return "REUSE.Software Analysis (forces *Ojo License Analysis*)";
+                            case "Check_agent_shagent":
+                              return "Software Heritage Analysis";
+                            default:
+                              return "";
```

Comment:
Can it be done similar to following messages?
https://github.com/fossology/FOSSologyUI/blob/main/src/constants/messages.js

Reply:
Yes it can be done. Will have to define constants for these values. Should I do that? @GMishx

Reply:
Done! @GMishx